cmake/SetupBoost.cmake

```diff
@@ -14,19 +14,3 @@ file(APPEND
 # Boost organizes targets as:
 # - Boost::boost is the header-only parts of Boost
 # - Boost::COMPONENT are the components that need linking, e.g. program_options
-
-# This is needed if your Boost version is newer than your CMake version
-# or if you have an old version of CMake (<3.5)
```

Comment:
I think you're misunderstanding the documentation because according to https://cmake.org/cmake/help/v3.8/module/FindBoost.html (the required new minimum) the targets are present.

Reply:
In the documentation you're linking it says

> If Boost was built using the boost-cmake project

and in my link to the newer version of the docs it says

> If Boost was built using the boost-cmake project or from Boost 1.70.0 on

So I understand that for Boost versions earlier than 1.70.0 one has to make sure to compile Boost with boost-cmake. My question was if we can rely on that

Reply:
I'm not relying on that. "The following IMPORTED targets are also defined:"